build.xml

```diff
@@ -161,4 +161,29 @@
               logoutput="true">
         </exec>
     </target>
+
+    <target name="deptrac:dump-png" description="Deptrac dump to png image">
+        <exec command="./bin/deptrac analyze --formatter-graphviz=1 --formatter-graphviz-dump-image=deptrac.png"
+              checkreturn="true"
+              logoutput="true">
+        </exec>
+    </target>
+    <target name="deptrac:dump-dot" description="Deptrac dump to dot file">
+        <exec command="./bin/deptrac analyze --formatter-graphviz=1  --formatter-graphviz-dump-dot=deptrac.dot"
```

Comment:
All report by console tools are stored in ${dir.reports}, there should be deptract director for it 
Deptrac have JUnit format report its be goo to add it hire, it be helpful in CI tools